compiler/test/fail_compilation/issue20704.d

```diff
@@ -4,13 +4,13 @@ fail_compilation/issue20704.d(17): Error: cannot modify constant `0`
 fail_compilation/issue20704.d(28): Error: template instance `issue20704.f2!int` error instantiating
 fail_compilation/issue20704.d(19): Error: cannot modify constant `0`
 fail_compilation/issue20704.d(30): Error: template instance `issue20704.f4!int` error instantiating
-fail_compilation/issue20704.d(17): Error: `S(0)` is not an lvalue and cannot be modified
 fail_compilation/issue20704.d(36): Error: template instance `issue20704.f2!(S)` error instantiating
-fail_compilation/issue20704.d(17): Error: `null` is not an lvalue and cannot be modified
 fail_compilation/issue20704.d(38): Error: template instance `issue20704.f2!(C)` error instantiating
```

Comment:
So this change would mean you would get `error instantiating` messages, but the reason for failure is suppressed? How is that useful?